sources/packages/api/src/services/archive-db/archive-db.service.spec.ts

```diff
@@ -0,0 +1,37 @@
+require("../../../env_setup");
+import { Test, TestingModule } from '@nestjs/testing';
+import * as dayjs from 'dayjs';
+import { ArchiveDbService } from './archive-db.service';
+
+describe('ArchiveDbService', () => {
+  let service: ArchiveDbService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [ArchiveDbService],
+    }).compile();
+
+    service = module.get<ArchiveDbService>(ArchiveDbService);
+    await service.init();
+  });
+
+  afterEach(async () => {
+    await service.connection.close();
+  })
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+
+  it('should init', async () => {
+    expect(service.connection).toBeDefined();
+  });
+
+  it('should get status', async () => {
+    const r: Array<{ permanent_disability_flg: string | null }> = await service.getIndividualPDStatus({ birthdate: dayjs("2000-03-11 00:00:00").toDate(), sin: '123456799' });
```

Comment:
Can you please elaborate on why we need to introduce the package "dayjs" to format the date here?

Reply:
1. This is to format day into yyyymmdd format, native Date object can do that but for dayjs it is much easy.
2. For date formatting we must need some utility, normally we use **moment**, dayjs is stripped out version for moment for example using traditional date object, we need to do something like this
```js
and it subtracts a day
    const options = {
      year: "numeric",
      month: "long",
      day: "numeric",
      timeZone: "UTC",
    };
    const formattedDate = new Date(date);
    return formattedDate.toLocaleDateString("en-CA", options);
    ```
with Moment or dayjs this whole things become single liner.